Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.7
+FROM python:3.6.6
```

Comment:
Why not just leave it at 3.6 (so when 3.6.7 comes out, there's no need for a code change)?